### DISCUSSION TITLE
Finding derivatives of a tensor in a flat space in index notation

### DISCUSSION THREAD

**Discussion by sunand22** [wxMaxima-developers/wxmaxima#1925] [Q&A]
Can Maxima be used to find the general indexed derivative of some Green's tensor, like for some simple tensor suppose : $\frac{x_i}{r^3}$
I want to find $$\partial_k \frac{x_i}{r^3}$$
I can define the metric as an identity I suppose. Any idea how to go forward from here?

**Comment by jIyajbe** [comment]
Maxima has three different ways of managing tensors; from your description, I suspect that itensor is the package you want. Check out https://maxima.sourceforge.io/docs/manual/maxima_130.html
